mslib/plugins/io/gpx.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.plugins.io.csv
+    ~~~~~~~~~~~~~~~~~~~~
+
+    plugin for csv format flight track export
+
+    This file is part of mss.
+
+    :copyright: Copyright 2008-2014 Deutsches Zentrum fuer Luft- und Raumfahrt e.V.
```

Comment:
in this line please add yourself as author.
there may be another line added :copyright: Copyright 2021 by the mss team, see AUTHORS.